pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "torch >= 1.10.0",
     "matplotlib >= 3.5.0",
     "wget >= 3.2",
+    "ipywidgets == 8.0.6",
```

Comment:
This dependency was needed for jupyter notebook support for sentence transformers progress bars